SD_DSC.adoc

```diff
@@ -3165,6 +3165,18 @@ When evaluating that the TOE has been provided and is labelled with a unique ref
 
 When evaluating the developer's coverage of the TOE in their CM system, the evaluator performs the work units as presented in the CEM.
 
+==== Basic Flaw Remediation (ALC_FLR.1) (optional)
+
+When evaluating the developer’s procedures regarding basic flaw remediation, the evaluator performs the work units as presented in the CEM.
```

Comment:
Very minor comment: you probably want to use ASCII ' instead of ’ here (and below).

Reply:
that's what I get for a straight copy-paste :)

Updated